index.Rmd

```diff
@@ -1,105 +1,299 @@
+
 ---
-title: "My Final Project Template"
-author: Your name here
-subtitle: Subtitle here if desired
+title: "Using R to Search for future mining and landing sites on Mars"
```

Comment:
Suggest dropping "Using R to" from the title.  Focus on the method, not the tool.